### PR TITLE
Add Github Action to automatically cleanup ubuntu-latest runners to fix runner running out of diskspace errors

### PR DESCRIPTION
**Pull request by apeltzer**
Fix for #2754

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

File: nf_core/pipeline-template/.github/workflows/ci.yml [hunk 1]
```diff
@@ -35,6 +35,9 @@ jobs:
         with:
           version: "{% raw %}${{ matrix.NXF_VER }}{% endraw %}"
 
+      - name: Cleanup Runner
```
Comment (ewels) [inline nf_core/pipeline-template/.github/workflows/ci.yml:38]:
```suggestion
      - name: Disk space cleanup
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +26,7 @@
 - Update python:3.11-slim Docker digest to 2a746e2 ([#2743](https://github.com/nf-core/tools/pull/2743))
 - Update actions/setup-python action to v5 ([#2739](https://github.com/nf-core/tools/pull/2739))
 - Update gitpod/workspace-base Docker digest to 45e7617 ([#2747](https://github.com/nf-core/tools/pull/2747))
+- Fix for #2754, disk space on gha runners ([#2755](https://github.com/nf-core/tools/pull/2755))
```
Comment (ewels) [inline CHANGELOG.md:29]: @mashehu  - it'd be nice if the changelog bot didn't add to the changelog if it already has edits in the PR.. :) 

```suggestion
```

**Review by ewels — APPROVED** [review]
Nice, thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (mashehu) [reply CHANGELOG.md]: it wouldn't have changed anything, if the added Changelog entry would have guessed the correct PR number 😉

**Comment by mashehu** [comment]
image

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 - Add empty line in README.md to fix badges. ([#2729](https://github.com/nf-core/tools/pull/2729))
 - Replace automatic branch detection in `nf-core download` CI test with hardcoded `dev` and input. ([#2727](https://github.com/nf-core/tools/pull/2727))
+- Add Github Action to automatically cleanup ubuntu-latest runners to fix runner running out of diskspace errors([#2754](https://github.com/nf-core/tools/issues/2754))
```
Comment (mashehu) [inline CHANGELOG.md:9]:
```suggestion
- Add Github Action to automatically cleanup ubuntu-latest runners to fix runner running out of diskspace errors([#2755](https://github.com/nf-core/tools/issues/2755))
```

Comment (ewels) [reply CHANGELOG.md]: 😱 

issue vs PR, both valid

**Comment by ewels** [comment]
image

File: CHANGELOG.md (hunk 3 above)
Comment (mashehu) [reply CHANGELOG.md]: ah, okay, we usually point to the PR, so I would go for that.

**Pull request merged**